.github/workflows/build-docker-images.yml

```
@@ -15,4 +15,13 @@ jobs:
       - uses: actions/checkout@v3
       - name: Build the full Docker image
-        run: docker build --file Dockerfile --tag streamlitapp:latest --output type=tar,dest=/dev/null . 
\ No newline at end of file
+        run: docker build --file Dockerfile --tag streamlitapp:latest --output type=tar,dest=/dev/null .
+
+  build-full-app-arm64:
+
+    runs-on: ubuntu-22.04-arm
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)